README.md

```diff
@@ -57,6 +57,10 @@ Supported methods:
     brew install ffmpeg cmake
     ```
 
+- Windows
+
+    no pre-requisites to be installed, skip this step

```

Comment:
Seems like redundant information to me. But if we are okay to keep it, I won't object it.

Reply:
@AjinkyaIndulkar i think this is not related to this PR right?

Reply:
Idk. Who added this?

Reply:
It was me

Reply:
I added this after the issue https://github.com/sensity-ai/dot/issues/44 to make it clear for Windows users.